### PR TITLE
Improved the UI of loklak.org/settings

### LINKED ISSUE #919: Improve the UI of loklak.org/settings
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Feature Description**
The UI of [https://loklak.org/settings](https://loklak.org/settings ) looks a bit outdated.
Improvement Needed.
settings-loklak
settings-loklak1
settings-lolak3

### PR DESCRIPTION
**Pull request by rajvaibhavdubey**
Improved the UI of the settings page, UI inspired from the site itself.

Fixes #1919

**Changes proposed in this pull request**
- The UI of the loklak.org/settings has been improved

**Screenshots** 
loklak-settings

**Closes #919**

### CODE REVIEW

**Review by simsausaurabh — CHANGES_REQUESTED** [review]
@rajvaibhavdubey Please check travis, there are some style issues.

**Comment by rajvaibhavdubey** [comment]
@simsausaurabh Could you please guide me about travis fixes, I'm not able to pinpoint the error

**Comment by simsausaurabh** [comment]
@rajvaibhavdubey Run command: ```ng lint loklak-search --format=stylish``` in your current project directory and fix the lint issues which shows up :+1:

**Comment by simsausaurabh** [comment]
@rajvaibhavdubey Please provide a test link to check your current changes.

File: src/app/settings/settings.component.scss [hunk 1]
```diff
@@ -1,48 +1,27 @@
...
+@media screen and (max-width:990px){
+	.image-banner {
+		height: 190px;
 	}
-}
+
+	.banner{
+		top:0;
+	}
+}
```
Comment (simsausaurabh) [inline src/app/settings/settings.component.scss:27]: Add EOF.

File: src/app/settings/settings.component.html [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
+
+  </div>
+  <hr>
+  <h3>Contact Us</h3>
+  <p>
+    If you would like to get in touch with us, you find our details on the
+    <a routerLink="/contact" target="_self">contact page</a>.
+  </p>
 </div>
+<app-footer></app-footer>
```
Comment (simsausaurabh) [inline src/app/settings/settings.component.html:35]: Add EOF.

File: src/app/settings/settings.component.spec.ts [hunk 3]
```diff
@@ -1,10 +1,52 @@
...
+	});
+	it('should have an app-footer component', async(() => {
+	const compiled = fixture.debugElement.nativeElement;
+
+		expect(compiled.querySelector('app-footer')).toBeTruthy();
+	}));
+
+	it('should have an app-navbar component', async(() => {
+	const compiled = fixture.debugElement.nativeElement;
+
```
Comment (simsausaurabh) [inline src/app/settings/settings.component.spec.ts:47]: Unnecessary line space.

File: src/app/settings/settings.component.spec.ts [hunk 4]
```diff
@@ -1,10 +1,52 @@
...
+			],
+			providers: [
+				{ provide: Store, useValue: {} },
+				{ provide: StateObservable, useValue: {} }
+			]
+		});
+	});
+	it('should have an app-footer component', async(() => {
+	const compiled = fixture.debugElement.nativeElement;
+
```
Comment (simsausaurabh) [inline src/app/settings/settings.component.spec.ts:41]: Unnecessary line space.

File: src/app/settings/settings.component.scss [hunk 5]
```diff
@@ -1,48 +1,27 @@
...
+.banner{
 	width: 100%;
-	// background-color: #f9f2f4;
-	position:  relative;
-	@include --center-flex();
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 0;
```
Comment (simsausaurabh) [inline src/app/settings/settings.component.scss:13]: Do we really require ```0``` values?

**Comment by simsausaurabh** [comment]
Closing this PR due to inactivity. Follow up once available.

**Pull request closed without merging**